src/Component/RequestBuilder/EdisMax.php

```diff
@@ -46,6 +46,7 @@ public function buildComponent($component, $request)
         $request->addParam('bf', $component->getBoostFunctions());
         $request->addParam('boost', $component->getBoostFunctionsMult());
         $request->addParam('uf', $component->getUserFields());
+        $request->addParam('sow', null === ($sow = $component->getSplitOnWhitespace()) ? null : ($sow ? 'true' : 'false'));
 
         return $request;
     }
```

Comment:
The new parameter should at least be described within docs/queries/select-query/building-a-select-query/building-a-select-query.md